src/components/Money/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { IMoney } from "@dashboard/utils/intl";
+
+import { Locale } from "../Locale";
+
+export const getMoney = (money: IMoney, locale: Locale) => {
```

Comment:
**suggestion:** I think we already have functions for that in index.ts file in this directory (`formatMoney`, `formatMoneyRange`). Look up these functions, perhaps they can be used.